package.json

```diff
@@ -31,7 +31,24 @@
     "onLanguage:dune-workspace",
     "onCommand:ocaml.select-sandbox",
     "onCommand:ocaml.open-terminal",
-    "onCommand:ocaml.open-terminal-select"
+    "onCommand:ocaml.open-terminal-select",
+    "onCommand:ocaml.open-terminal-select",
```

Comment:
This seems to be an unintentional duplication.

Reply:
Indeed, thanks!